web/source/kmwcallback.js

```diff
@@ -54,7 +54,7 @@
         Lelem._KeymanWebSelectionEnd=Le;
         keymanweb._IgnoreNextSelChange = 0;
         if(Ptext!=null)keymanweb.KO(0, Lelem, Ptext);
-        if(typeof(PdeadKey)!='undefined') keymanweb.KDO(0, Lelem, PdeadKey);
+        if((typeof(PdeadKey)!=='undefined') && (PdeadKey !== null)) keymanweb.KDO(0, Lelem, PdeadKey);
```

Comment:
It threw a warning here, absolutely.  It believed that the type of Pdeadkey could be undefined/null in the function call and threw a warning there.  If keymanweb.KDO truly isn't allowed to take a null as an argument, we need this.